LayoutTests/media/media-can-play-vp8.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="video-test.js"></script>
+        <script>
+            function start() {
+                video = mediaElement = document.getElementsByTagName('video')[0];
+
+                if (window.internals) {
+                    // Disable VP9 for testing on macOS to confirm VP8 still works.
+                    internals.setHardwareVP9DecoderDisabledForTesting(true);
+                }
+
+                testExpected("video.canPlayType('video/webm')", "maybe");
+                testExpected("video.canPlayType('video/webm; codecs=vp8')", "probably");
```

Comment:
Since this test is called `media-can-play-vp8` it might make sense to re-enable the hardware VP9 decoder after the first expectations run, and then run the same expectations again to make sure that the VP9 decoder doesn't change behaviour.

Furthermore, since this test won't be able to pass in its current state until #1935 lands, I would recommend renaming the test to something more specific (relating to the VP9 decoder + VP8 canPlayType and since we already have multiple tests checking canPlayType for most WebM codecs).

Instead of checking whether `maybe` and `probably` are returned, we can check that the value of  `video.canPlayType('video/webm')` BEFORE the VP9 decoder is disabled is equal to `video.canPlayType('video/webm')` AFTER the VP9 decoder is disabled and it should be able to pass in its current state.